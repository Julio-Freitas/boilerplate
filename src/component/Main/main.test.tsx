import { render, screen } from '@testing-library/react';
import { Main } from '.';

describe('<Main', () => {
  it('should be render component', () => {
    const { container } = render(<Main />);
    expect(container.firstChild).toMatchSnapshot();
    expect(
      screen.getByRole('heading', { name: 'ola main' })
    ).toBeInTheDocument();
  });
});
