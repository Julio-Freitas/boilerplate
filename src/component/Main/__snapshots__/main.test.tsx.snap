// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main should be render component 1`] = `
<div>
  <h1>
    ola main
  </h1>
</div>
`;
